lib/feature/setting/presentation/pages/setting_page.dart

```diff
@@ -0,0 +1,101 @@
+}
+
+/// This is the private State class that goes with MyStatefulWidget.
+class _SettingPageState extends State<SettingPage> {
+  String dropdownValue = '10';
+  int count = 1;
+  List<bool> _values = [true];
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppBar(),
+      body: Row(
+        children: [
+          Column(
+            children: [
+              Column(
+                children: [
+                  Container(
+                    padding: EdgeInsets.all(32.0),
+                    child: FlatButton(
+                      onPressed: () => Navigator.of(context).pop(),
+                      child: Row(
+                        children: [
+                          Icon(Icons.fast_rewind),
+                          Text(
+                            'SETTINGS',
+                            style: TextStyle(
+                              fontSize: 32,
+                              fontWeight: FontWeight.w900,
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                  ),
+                  Row(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.all(24.0),
+                        child: Container(
+                          child: Text("NUMBER OF QUESTION"),
+                        ),
+                      ),
+                      DropdownButton<String>(
+                        value: dropdownValue,
+                        elevation: 16,
+                        style: TextStyle(color: Colors.deepPurple),
+                        underline: Container(
+                          height: 2,
+                          color: Colors.deepPurpleAccent,
+                        ),
+                        onChanged: (String newValue) {
+                          setState(() {
+                            dropdownValue = newValue;
+                          });
+                        },
+                        items: <String>['10', '20', '30', '40']
+                            .map<DropdownMenuItem<String>>((String value) {
+                          return DropdownMenuItem<String>(
+                            value: value,
+                            child: Text(value),
+                          );
+                        }).toList(),
+                      )
+                    ],
+                  ),
+                  Row(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.all(24.0),
+                        child: Container(
+                          child: Text("ONLY WEEK QUESTION"),
+                        ),
+                      ),
+                      Switch(
+                        value: _values[0],
+                        onChanged: (value) {
+                          setState(() {
```

Comment:
same as above:

in clean architecture, you don't need to use setState any more, whole this stuff should be moved to the bloc. Bloc is responsible for any logic in your page. The `page` should be a dummy and be responsible only on widgets and how to display them